akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1348,8 +1348,8 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
    * @param initial      The initial element to be emitted, in case upstream is able to stall the entire stream.
    * @see [[#expand]]    for a version that can overwrite the original element.
```

Comment:
One more thing I noticed today - this does not render well within JavaDoc - for the Java API versions, I'm going to move the description portion to the main body of the doc (as suggested by [this related SO answer](https://stackoverflow.com/questions/5011291/usage-of-see-in-javadoc#5014784)), and leave `@see` with just the method names. This is because the only other option would be to use the fragile, "raw" `a` link.

Reply:
👍